onnxruntime/contrib_ops/cpu/bert/bias_gelu_fusion.cc

```diff
@@ -18,28 +19,14 @@ ONNX_OPERATOR_KERNEL_EX(
 
 template <typename T>
 Status BiasGelu<T>::Compute(OpKernelContext* ctx) const {
-  const Tensor* X = ctx->Input<Tensor>(0);
-  const auto input_dims = X->Shape().GetDims();
-  if (input_dims.size() < 1) {
-    return ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_ARGUMENT,
-                           "Shape of Input 0 is expected to have at least 1 dimension, got ", input_dims.size());
-  }
+  ORT_RETURN_IF_ERROR(fast_gelu::CheckInputs(ctx));
```

Comment:
>fast_gelu [](start = 22, length = 9)

Extract a base class for BiasGelu and FastGelu, and put this function as a method of the base class?

Reply:
It is also used by FastGelu cuda class. Move it to a helper function could avoid that CUDA class depends on CPU class.